poker/utils/constants.py

```diff
@@ -1,22 +1,40 @@
 from enum import Enum
 
+import emoji
 
-SUITS = ["hearts", "diamonds", "spades", "clubs"]
-FACE_CARDS = "J Q K A".split()
+SUITS = f"{emoji.emojize(':heart_suit:')} {emoji.emojize(':diamond_suit:')} {emoji.emojize(':spade_suit:')} {emoji.emojize(':club_suit:')}".split()
 NUMBER_CARDS = list(range(2, 11))
-INCREMENT_LIMIT = 20
+INCREMENT_LIMIT = 100
 SINGLE_CHIP = 1
 DOUBLE = 2
+COMPETITION = (list(range(1, 5)))
```

Comment:
why wrap in `()` here?

Reply:
Good catch. I'm not sure why I did that. I'll remove it.